library/core/src/intrinsics.rs

```diff
@@ -1901,11 +1901,21 @@ pub unsafe fn copy_nonoverlapping<T>(src: *const T, dst: *mut T, count: usize) {
 /// ```
 /// use std::ptr;
 ///
+/// /// # Safety:
```

Comment:
This `Safety` is already there above, this feels duplicated, two `Safety` section is too safe to be safe, I think one `Safety` should be enough.